[IMP] Inventory: FEFO removal tracking update (#17633)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/fefo.rst

Reviewer: ```suggestion
Serial Numbers`. After :guilabel:`Lots & Serial Numbers` is enabled, the :guilabel:`Expiration Dates`
```

Reviewer: ```suggestion
Configuration
```

Reviewer: ```suggestion
Set removal strategies
```

-----------------------------------------

Next, ensure the intended product is tracked by lots or serial numbers by navigating to the product
form through :menuselection:`Inventory app --> Products --> Products` and selecting the desired
product. On the product form, open the :guilabel:`General Information` tab. In the
:guilabel:`Tracking` field, select either :guilabel:`By Unique Serial Number` or :guilabel:`By
Lots`.

.. image:: fefo/track-by-lots.png
   :alt: Enable tracking by lot.

> Setting dates

Reviewer: ```suggestion
Set FEFO dates
```

-------------

Next, set the dates for the product. In the :guilabel:`Inventory` tab of the product form, under the
:guilabel:`Traceability` heading, select the :guilabel:`Expiration Date` checkbox. The
:guilabel:`Dates` heading appears. Specify the :guilabel:`Expiration Date`, :guilabel:`Best Before
Date`, :guilabel:`Removal Date`, and :guilabel:`Alert Date` settings for the product. See
:ref:`Removal date <inventory/warehouses_storage/removal-date>` for more information.

> Assigning lots and serial numbers

Reviewer: ```suggestion
Assign lots and serial numbers
```

Products **must** be removed from inventory before their *removal date*, which is set as a certain
number of days before the product's *expiration date*.

Before setting this date, ensure that :guilabel:`Tracking` :guilabel:`By Lots` or :guilabel:`By

> Unique Serial Number` is enabled on the product form.

Reviewer: ```suggestion
Before setting this date, ensure that the :guilabel:`Tracking` property field is set to :guilabel:`By Lots` or :guilabel:`By
Unique Serial Number`.
```
